src/dev/src/app/exportable-datagrid/exportable-datagrid.module.ts

```diff
@@ -0,0 +1,21 @@
+import { NgModule } from '@angular/core';
+import { ExportableDatagridComponent } from './exportable-datagrid.component';
+import { RouterModule } from '@angular/router';
+import { ClrDatagridModule, ClrDropdownModule } from '@clr/angular';
+import { NgForOf } from '@angular/common';
+import { ExportDatagridButtonModule } from '../../../../clr-addons/export-datagrid-button';
```

Comment:
this is unnecessarily complicated, it should be something like ../export-datagrid-button